csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1146,6 +1153,91 @@ Examples:
+5. Patch version Z (x.y.Z | x > 0) MUST be incremented if only backwards compatible bug fixes are introduced. A bug fix is defined as an internal change that fixes incorrect behavior.
+
+   > In the context of the document this is the case  e.g. for spelling mistakes.
+
+6. Minor version Y (x.Y.z | x > 0) MUST be incremented if the content of an existing element changes except for those which are covert through rule 7. It MUST be incremented if substantial new information are introduced or new elements are provided. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.
+7. Major version X (X.y.z | X > 0) MUST be incremented if a new comparison with the end user's asset database is required. This includes:
+
+   * changes (adding, removing elements or modifying content) in `/product_tree` or elements which contain `/product_tree` in their path
+   * adding or removing items of `/vulnerabilities`
+   * adding or removing elements in:
+     * `/vulnerabilities[]/product_status/first_affected`
+     * `/vulnerabilities[]/product_status/known_affected`
+     * `/vulnerabilities[]/product_status/last_affected`
+   * removing elements from:
+     * `/vulnerabilities[]/product_status/first_fixed`
+     * `/vulnerabilities[]/product_status/fixed`
+     * `/vulnerabilities[]/product_status/known_not_affected`
+
+   It MAY also include minor and patch level changes. Patch and minor version MUST be reset to 0 when major version is incremented.
+8. A pre-release version (document status `draft`) MAY be denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version. Identifiers MUST comprise only ASCII alphanumerics and hyphens [0-9A-Za-z-]. Identifiers MUST NOT be empty. Numeric identifiers MUST NOT include leading zeroes. Pre-release versions have a lower precedence than the associated normal version. A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version. Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7, 1.0.0-x.7.z.92, 1.0.0-x-y-z.–
+9. Build metadata MAY be denoted by appending a plus sign and a series of dot separated identifiers immediately following the patch or pre-release version. Identifiers MUST comprise only ASCII alphanumerics and hyphens [0-9A-Za-z-]. Identifiers MUST NOT be empty. Build metadata MUST be ignored when determining version precedence. Thus two versions that differ only in the build metadata, have the same precedence. Examples: 1.0.0-alpha+001, 1.0.0+20130313144700, 1.0.0-beta+exp.sha.5114f85, 1.0.0+21AF26D3—-117B344092BD.
+10. Pre-release and build metadata SHOULD NOT be included if the `/document/status` is `final`.
+11. Precedence refers to how versions are compared to each other when ordered.
```

Comment:
I'd say items 8, 9, and 11 should just refer to the semver documentation instead of being copy-pasted here. E.g.:

```markdown
8. A pre-release version of a CSAF document (document status `draft`) may be denoted by
appending a hyphen and and a series of dot separated identifiers immediately following the
patch version, as specified by the Semantic Versioning Specification (https://semver.org/#spec-item-9).
```

Reply:
I am +0 on providing the copy as a service to the reader. I once started to support the GeoJSON community to make their well-working but informal  specification into an RFC so as to allow OASIS specifications to refer to them normatively. To semver.org we can only refer informally, and shall provide the content as a copy. This makes the future OASIS standard stable and the reference serves still the purpose of indicating the source (no plagiarism)

Reply:
Fair enough, if we feel it's better to include the content as is to be more future-proof I'm fine with that.

Reply:
@sthagen: Thanks for the comment. I moved [SemVer] from the "Normative References" to the "Informative Reference". Could you please check that I got the right format for OASIS?

Reply:
@sthagen: Thank you.